plenum/server/replica.py

```diff
@@ -452,6 +453,8 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
 
         HookManager.__init__(self, ReplicaHooks.get_all_vals())
 
+        self.consensus_provider = ConsensusDataProvider(self.name)
```

Comment:
`ConsensusDataProvider` expects node name in constructor, not replica name

Reply:
ok